unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -62,6 +62,17 @@ public bool IsVisible
 		protected float[] headingAngles = new float[] { 0.0f, 90.0f, 180.0f, 270.0f };
 		protected float[] horizonAngles = new float[] { 60.0f, 30.0f, 0.0f, 330.0f };
 
+		//When passing in the TeleportFull action's horizon value, these are the only values that should be allowed
+		//they will be translated to the actual horizonAngles used above, as 30 should rotate the agent's camera 30 degrees up, but actually the
+		//value 330 or -30 about the x axis is what will angle the camera up... so yeah this is just to make it less confusing for the user
+		protected Dictionary<int, float> validHorizonAngleValues = new Dictionary<int, float>()
+		{
+			{30, -30.0f},
+			{0, 0.0f},
+			{-30, 30.0f},
+			{-60, 60.0f},
+		};
+
```

Comment:
I think it makes more sense for a horizon of 30 to indicate a rotation of 30 degrees downwards rather than having to do this conversion to do the opposite. I also worry that this breaks backwards compatibility.

Reply:
hmm, maybe there is a better way to structure this. Basically I was trying to address this post: https://github.com/allenai/ai2thor/issues/154

Reply:
Ah, I would just mark #154 as #wontfix as this isn't a bug, if anything I would just make the documentation clearer.